test/index.spec.ts

```diff
@@ -810,9 +813,9 @@ describe('onPostBuild', () => {
       },
     ])
 
-    const oldLog = console.log
+    const oldLoggerInfo = logger.info
```

Comment:
I left this as was but we could probably just spy on the `logger.info` here to confirm it gave the message we expect.